app/models/PacketTypes.js

```diff
@@ -77,7 +77,7 @@ var PacketTypes = {
     'uhf_channel_status':  ['isPositiveNumber', 'isInteger'],
     'waypoint_index': ['isPositiveNumber', 'isInteger'],
     'waypoint_count': ['isPositiveNumber', 'isInteger'],
-    'following_path': 'isInteger'	
+    'following_path': 'isInteger'   
```

Comment:
remove the space at the end if you're going to modify that line (green means you added blank space)